common-npm-packages/packaging-common/universal/ArtifactToolUtilities.ts

```diff
@@ -55,13 +55,20 @@ export async function getArtifactToolFromService(serviceUri: string, accessToken
 
     // https://github.com/nodejs/node-v0.x-archive/issues/2862
     if (arch === "ia32") {
-        if (process.env.PROCESSOR_ARCHITEW6432 != null && process.env.PROCESSOR_ARCHITEW6432.toUpperCase() === "AMD64") {
-            arch = "amd64";
-        }
-        else if ((process.env.PROCESSOR_ARCHITEW6432 == null && process.arch.toUpperCase() === "ARM64") ||
-                (process.env.PROCESSOR_ARCHITEW6432 != null && process.env.PROCESSOR_ARCHITEW6432.toUpperCase() == "ARM64")) {
-                arch = "arm64";
-            }
+        if (process.env.PROCESSOR_ARCHITEW6432 != null)
```

Comment:
Please follow our existing code style in both files -- 4 space indent, no tab characters, opening brace on same line